index.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
     "Registered students can access course details via [StudIP](https://studip.uni-rostock.de/dispatch.php/course/overview?cid=1df474b51c0813ffdeb34be5e66ef960) and [LSF](https://lsf.uni-rostock.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=78325&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung).\n",
     "\n",
-    "The lectures/exercises are held each summer semester, starting in 2015.\n",
+    "The lectures/exercises are held each summer semester, starting in 2015. Note, that we are currently (summer term 2022) modifying the notebooks to work with recent Python...\n",
```

Comment:
I think this additional sentence isn't helping anyone. And recent Python isn't really the problem, it's rather all the third-party packages, isn't it?

Reply:
True indeed, it is not Python itself but rather libs. Removed.